models/public/i3d-rgb-tf/i3d-rgb-tf.md

```diff
@@ -73,15 +75,17 @@ Channel order is `RGB`.
 
 ### Original Model
 
-Action classifier according to Kinetics-400\* action classes, name - `Softmax`, shape - `1,400`, format is `B,C`, where:
+Action classifier according to Kinetics-400\* action classes, name - `Softmax`, shape - `1, 400`, format is `B, C`, where:
```

Comment:
```suggestion
Action classifier according to Kinetics-400 action classes, name - `Softmax`, shape - `1, 400`, format is `B, C`, where:
```
Also, what about adding link here, as for COCO?

Reply:
Done